src/styles/card.scss

```diff
@@ -0,0 +1,155 @@
+@import './variables';
+@import './mixins';
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @include on-mobile {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @include on-tablet {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @include on-desktop {
+    grid-template-columns: repeat(4, 200px);
+  }
```

Comment:
Your deployment does not seem to be up to date with your code:

Deployed site:
![image](https://github.com/mate-academy/layout_catalog/assets/52002856/cb4cead0-f94f-44a9-9b34-dd54b7e04c38)

![image](https://github.com/mate-academy/layout_catalog/assets/52002856/c227a002-60ba-4d52-a048-22cc57db8106)

Reply:
Gonna check this, thank's!